app/src/lib/stores/app-store.ts

```diff
@@ -2729,7 +2732,9 @@ export class AppStore extends TypedBaseStore<IAppState> {
     repository: Repository,
     name: string,
     startPoint: string | null,
-    uncommittedChangesStrategy: UncommittedChangesStrategy = UncommittedChangesStrategy.askForConfirmation
+    uncommittedChangesStrategy: UncommittedChangesStrategy = {
```

Comment:
There are a few other places where we use an `UncommitedChangesStrategy` object literal, is it worth pulling this off into a constant to clean up this function signature?